schema/InlineContent.schema.yaml

```
@@ -20,3 +20,4 @@ anyOf:
   - $ref: Strong
   - $ref: Subscript
   - $ref: Superscript
+  - $ref: CiteGroup
```

Based on the example in `Cite.md` and the HTML [`cite` tag](url) believe `Cite` should also be added to InlineContent here, as well as the typeMap.
```suggestion
  - $ref: Cite
  - $ref: CiteGroup
```

---

Done in 0113559.